core-ui/src/components/Extensibility/components/Plain.js

```diff
@@ -6,7 +6,7 @@ export function Plain({ value, structure, schema }) {
   return (
     <div>
       {structure.children?.map(def => (
-        <Widget value={value} structure={def} schema={schema} />
+        <Widget value={value} structure={def} schema={schema} key={def} />
```

Comment:
This won't do - since `def` is an object, the key would be always `[Object object]`.